server/protocols/webadmin/webadmin-mailrepository/src/test/java/org/apache/james/webadmin/routes/MailRepositoriesRoutesTest.java

```diff
@@ -217,8 +217,11 @@ public void listingKeysShouldApplyLimitAndOffset() throws Exception {
             .name("name3")
             .build());
 
-        when()
-            .get(MY_REPO_MAILS + "?offset=1&limit=1")
+        given()
+            .param("limit", "1")
```

Comment:
ok, I was wrong, it's queryParam, not param.
You may wonder if you really test what you think

Reply:
I checked and both seems to do the same (even checked with wireshark). What's the difference?